src/app/Topology/Shared/utils.tsx

```diff
@@ -41,26 +50,41 @@ export const getStatusTargetNode = (node: TargetNode | EnvironmentNode): [NodeSt
       : [
           NodeStatus.warning,
           {
-            title: 'Failed to compute JVM ID',
+            title: 'Failed to generate the JVM identifier',
             description: (
               <>
-                <Text component={TextVariants.p}>
-                  If target has JMX Authentication enabled, add the credential to Cryostat keyring.
-                </Text>
-                <Text component={TextVariants.p}>
-                  If the target has SSL enabled over JMX, add its certificate to Cryostat truststore.
-                </Text>
+                <Text component={TextVariants.p}>Check the target authentication settings:</Text>
               </>
             ),
             callForAction: [
-              <WarningResolverAsCredModal key={`${node.target.alias}-resolver-as-credential-modal`}>
-                <Button variant="link" isSmall style={{ padding: 0 }}>
-                  Add credentials
-                </Button>
-              </WarningResolverAsCredModal>,
-              <WarningResolverAsLink key={`${node.target.alias}-resolver-as-link-to-security`} to="/security">
-                Add certificates
-              </WarningResolverAsLink>,
+              <Text key="jmx-auth" component={TextVariants.p}>
+                If{' '}
+                <DescriptionListTermHelpText>
+                  <Popover maxWidth="40rem" headerContent="JMX Authentication" bodyContent={<JmxAuthDescription />}>
+                    <DescriptionListTermHelpTextButton>JMX Authentication</DescriptionListTermHelpTextButton>
+                  </Popover>
+                </DescriptionListTermHelpText>{' '}
+                is enabled,{' '}
+                <WarningResolverAsCredModal key={`${node.target.alias}-resolver-as-credential-modal`}>
+                  <Button variant="link" isSmall style={{ padding: 0 }}>
+                    add credentials
+                  </Button>
+                  .
+                </WarningResolverAsCredModal>
+              </Text>,
+              <Text key="jmx-ssl" component={TextVariants.p}>
+                If{' '}
+                <DescriptionListTermHelpText>
+                  <Popover maxWidth="40rem" headerContent="JMX over SSL" bodyContent={<JmxSslDescription />}>
```

Comment:
JMX over SSL -> SSL enabled for JMX?

Reply:
I think it works well as part of the call-to-action text items but I'm not sure if that flows so well as the header/title for the popover. That title appears in two places:

![image](https://github.com/cryostatio/cryostat-web/assets/3787464/10fc9578-7111-4e93-9b7a-c4490d61d528)

![image](https://github.com/cryostatio/cryostat-web/assets/3787464/d6a3fcd6-0efa-49d5-b41c-ff64973dbc66)